.prettierignore

```diff
@@ -1,3 +1,4 @@
 pnpm-lock.yaml
+.astro
```

Comment:
Shouldn't `.astro` files be gitignored?

Reply:
The [`.astro` folder](https://github.com/stackblitz/tutorialkit/blob/db233c7b37a787677d205586a88663913e71e99f/packages/template/.gitignore#L2) is ignored already, I noticed that prettier was giving a warning locally because it doesn't ignore it.

It doesn't produce that warning on CI because we run prettier before running the build script. If the order was reversed, the CI would fail which would be a bit silly.

Reply:
OOh weird. Ok, then it's fine.